MVP.md

```diff
@@ -1,27 +1,28 @@
 # RAP Benefits
 
 A RAP should:
-- Improve the quality of the statistics.
-- Increase trust in the statistics by producers, their managers and users.
+- Improve the quality of the analysis.
+- Increase trust in the analysis by producers, their managers and users.
 - Create a more efficient process.
 - Improve business continuity and knowledge management.
 
 # RAP MVP
 
 In order to achieve these benefits, at a minimum a RAP must:
+- Augment extant technical and quality assurance processes with **peer review** to ensure that the process is reproducible and that the below requirements have been met.
 - **Minimise manual steps**, for example copy-paste, point-click or drag-drop steps. Where it is absolutely necessary to include a manual step in the process this must be documented as described below. 
-- Be built using **open source software** which is available to anyone.
-- Guarantee an **audit trail** using version control software.
+- Be built using software where the code is available to view internally, but preferably using **open source software** which is available to anyone.
```

Comment:
My understanding of the discussion last week was that there was agreement that open source software should be a requirement?

Reply:
I was under the impression it was slightly more nuanced, in that the code should be able to be seen by people internally, and open source if possible. For example, we have many engagement/mission/campaign level simulation models which are bespoke and can be quite sensitive.

Reply:
@jl5000 So would you see those engagement/mission/campaign models as being an example of RAP, even though they're not built using open source software? I agree with @annahprice and think open source software should be a requirement but happy to be challenged on that.

Reply:
Yes. For example, we have models that have sensitive engagement tactics within the code. There's no way we could make that open source, but we could make it open within Dstl. I'm not sure we want to indicate people can't have a MVP just because they're working on classified projects.

Reply:
I think we might be having a cross conversation! This bullet point is about using open source software, rather than opening up the code - presumably it's ok to specify that open source software (e.g. R or python) should be used no matter what? And then on the issue of publishing the code, the caveat below should cover instances where it's not appropriate to share publicly.

Reply:
Ah, sorry, my bad. You can ignore this change then.